src/test/regress/expected/mixed_relkind_tests.out

```diff
@@ -0,0 +1,932 @@
+ Custom Scan (Citus Adaptive)
+   Task Count: 4
+   Tasks Shown: One of 4
+   ->  Task
+         Node: host=localhost port=xxxxx dbname=regression
+         ->  WindowAgg
+               ->  Sort
+                     Sort Key: foreign_distributed_table.a
+                     ->  Foreign Scan on foreign_distributed_table_1513022 foreign_distributed_table
+(9 rows)
+
+-- pull to coordinator WINDOW
+EXPLAIN (COSTS OFF) SELECT a, COUNT(*) OVER (PARTITION BY a+1) FROM partitioned_distributed_table;
+                                                    QUERY PLAN
+---------------------------------------------------------------------
+ WindowAgg
+   ->  Sort
+         Sort Key: remote_scan.worker_column_2
+         ->  Custom Scan (Citus Adaptive)
+               Task Count: 4
+               Tasks Shown: One of 4
+               ->  Task
+                     Node: host=localhost port=xxxxx dbname=regression
+                     ->  Append
+                           ->  Seq Scan on partitioned_distributed_table_1_1513014 partitioned_distributed_table_1
+                           ->  Seq Scan on partitioned_distributed_table_2_1513018 partitioned_distributed_table_2
+(11 rows)
+
+EXPLAIN (COSTS OFF) SELECT a, COUNT(*) OVER (PARTITION BY a+1) FROM foreign_distributed_table;
+                                             QUERY PLAN
+---------------------------------------------------------------------
+ WindowAgg
+   ->  Sort
+         Sort Key: remote_scan.worker_column_2
+         ->  Custom Scan (Citus Adaptive)
+               Task Count: 4
+               Tasks Shown: One of 4
+               ->  Task
+                     Node: host=localhost port=xxxxx dbname=regression
+                     ->  Foreign Scan on foreign_distributed_table_1513022 foreign_distributed_table
+(9 rows)
+
+-- FOR UPDATE
+SELECT * FROM partitioned_distributed_table WHERE a = 1 ORDER BY 1,2 FOR UPDATE;
+ a | b
+---------------------------------------------------------------------
+ 1 | 2
+(1 row)
+
+SELECT * FROM unlogged_distributed_table WHERE a = 1 ORDER BY 1,2 FOR UPDATE;
+ a | b
+---------------------------------------------------------------------
+ 1 | 2
+(1 row)
+
+VACUUM partitioned_distributed_table;
+TRUNCATE partitioned_distributed_table;
+-- drop column followed by SELECT in transaction block
+BEGIN;
+  ALTER TABLE partitioned_distributed_table DROP COLUMN b CASCADE;
+NOTICE:  drop cascades to 2 other objects
+DETAIL:  drop cascades to view view_on_part_dist
+drop cascades to materialized view mat_view_on_part_dist
+  SELECT * FROM partitioned_distributed_table;
+ a
+---------------------------------------------------------------------
+(0 rows)
+
+COMMIT;
+BEGIN;
+  ALTER TABLE foreign_distributed_table DROP COLUMN b CASCADE;
+  SELECT * FROM foreign_distributed_table;
+ a
+---------------------------------------------------------------------
+(0 rows)
+
+COMMIT;
+-- cleanup at exit
+DROP SCHEMA mixed_relkind_tests CASCADE;
```

Comment:
when there are so many objects, they sometimes lead to flaky tests at DROP. Maybe set the log level to error?